homeassistant/components/heos/__init__.py

```diff
@@ -337,8 +347,8 @@ def connect_update(self):
         async def _async_handle_player_added():
             await self.async_update_groups(SIGNAL_HEOS_PLAYER_ADDED)
 
-        self._hass.helpers.dispatcher.async_dispatcher_connect(
-            SIGNAL_HEOS_PLAYER_ADDED, _async_handle_player_added
+        async_dispatcher_connect(
```

Comment:
Do we need to disconnect this signal when reloading the config entry?

Reply:
Is `async_unload_entry` called on reaload? In this case I could just disconnect the signal right there.

Reply:
Alright, the signal is now disconnected in `disconnect_update()` through `entry.async_on_unload()`